tests/php/sync/test_class.jetpack-sync-options.php

```diff
@@ -198,6 +198,7 @@ public function test_sync_default_options() {
 			'wordads_display_page'                 => '1',
 			'wordads_display_archive'              => '1',
 			'wordads_custom_adstxt'                => 'pineapple',
+			'site_user_type'                       =>  json_encode( array( 0 => 'pineapple' ) ),
```

Comment:
Let's use `wp_json_encode()` here instead. Also, `0` isn't a valid user ID - lets use a positive integer instead.

Reply:
Nit: There is an unnecessary space before the `json_encode()` here.

Reply:
Addressed.